blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Dasharo compatible with MSI PRO Z690-A release v1.1.1
+abstract: 'A short overview of changes introduced in v1.1.1
+           release of Dasharo compatible with MSI PRO Z690-A
+	   with a short demo of Qubes OS 4.1.2.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2023-02-24
+archives: "2023"
+
+tags:
+  - firmware
+  - coreboot
+  - Dasharo
+categories:
+  - Firmware
+
+
+---
+
+# Introduction
+
+New year and new exciting news. Dasharo Team has just released a new version of
+Dasharo firmware compatible with MSI PRO Z690-A. Last v1.1.0 version was
+released on 22th of November 2022 and after 3 months the time has come for v1.1.1.
+Let's see what's new.
+
+# Key changes in new release
+
+The past 3 months were mostly spent on debugging and resolving issues reported
+by community on Matrix and
+[GitHub](https://github.com/Dasharo/dasharo-issues/issues/new/choose). By the
+way, if you haven't joined our [Matrix Dasharo space](https://matrix.to/#/#dasharo:matrix.org)
+yet, do it quick! That is where all interesting stuff happens, like
+various technical discussions, hardware setups, firmware configurations and
+encountered issues.
```

Comment:
There are already 100+ users in General channel.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/513/commits/0f29e7a980e1814bfdf6c3d8ab045c7ba6d63782#diff-b67e40bb2c6780be37b7e6cf6fc2a2a9f9ad977388795cc0c5bdfa6576dc52b6R43